ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -728,6 +761,29 @@ void setChunkException(RouterException exception) {
     }
 
     /**
+     * Slice this chunk's data to only include the bytes within the operation's specified byte range.
+     * @param blobData the {@link BlobData} for this chunk.
+     * @return A {@link ByteBuffer} that only includes bytes within the operation's specified byte range.
+     */
+    ByteBuffer filterChunkToRange(BlobData blobData) {
```

Comment:
I know it is the same package, but if your intent is that this function could be useful in sub classes, wouldn't making it `protected` better than package private?